x-pack/plugins/saved_objects_tagging/public/services/tags/tags_client.ts

```diff
@@ -90,11 +113,17 @@ export class TagsClient implements ITagInternalClient {
   }
 
   public async getAll({ asSystemRequest }: GetAllTagsOptions = {}) {
+    const startTime = window.performance.now();
     const fetchOptions = { asSystemRequest };
     const { tags } = await this.http.get<{ tags: Tag[] }>(
       '/api/saved_objects_tagging/tags',
       fetchOptions
     );
+    const duration = window.performance.now() - startTime;
+    reportPerformanceMetricEvent(this.analytics, {
+      eventName: GET_ALL_TAGS_EVENT,
+      duration,
+    });
```

Comment:
Do we really want to compute the duration from the browser-side (and therefor include the server request's delay)? I would have done this on the server-side, at the route handler's level (but I don't know how it's done for other performance journeys, so if that's how we decided to do it, it's fine with me)

Reply:
I see your point. We agreed not use EBT on the sever side, but rely on APM data there. Still we are adding custom EBT events on client(browser)-side in journeys to track time-to-data (and sometimes UI rendering) as it is super easy to monitor values change and fo down to APM only in case of regression investigation.

Reply:
Sounds good, thanks for explaining!